src/components/music/NoSongsMessage.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useCallback, useState } from "react";
+import { useDispatch } from "react-redux";
+
+import settingsActions from "../../store/features/settings/settingsActions";
+import { SettingsState } from "../../store/features/settings/settingsState";
+import { Button } from "../ui/buttons/Button";
+
+export const NoSongsMessage = () => {
+  const dispatch = useDispatch();
+  const [showMessage, setShowMessage] = useState(false);
+
+  const editSettings = useCallback(
+    (patch: Partial<SettingsState>) => {
+      dispatch(settingsActions.editSettings(patch));
+    },
+    [dispatch]
+  );
+
+  return (
+    <>
+      <h2>No songs found</h2>
```

Comment:
Can you move all this text into en.json for localising?

Reply:
✅ Done!